src/apis/user.js

```diff
@@ -6,7 +6,15 @@ const UserAPI = {
       .post(`${ process.env.REACT_APP_AMS_API_BASE }/users/token`)
       .set('Accept', 'application/json')
       .send(params);
+  },
+  signup: (params) => {
+  	return request
+  		.post('http://spree-ams.herokuapp.com/api/v1/users')
```

Comment:
Do not hard-code the URL, tokens and anything else. Use `.env` file for the same.